examples/register-chain/app.ts

```diff
@@ -8,7 +8,12 @@ export const start = async () => {
 
     await chainRegistry.registerChain({
       name: 'MyChain',
-      urls: ['http://geth:8545']
+      urls: ['http://geth:8545'],
+      listener: {
```

Comment:
I forgot to add `listener` attributes in my previous MR about ChainRegistry wrapper therefore I am covering this here